package.json

```diff
@@ -61,7 +61,8 @@
   },
   "dependencies": {
     "@prisma/client": "4.8.1",
-    "prisma": "4.8.1"
+    "prisma": "4.8.1",
+    "react-colorful": "^5.6.1"
```

Comment:
Always install with a pinned version tag (no `^`), except @types/X.

So `npm install -E react-colorful` or `npm install -E -D react-colorful`. This works better with our upgrade approach (explicitly upgrade versions, not `npm update`).

-> Remove `^` and run `npm install` to regenerate lockfile

Reply:
Also, `react-colorful` is a dependency of `apps/frontend-manage`, not the root project (this would break in builds or production)! Remove from the root `package.json` and either `cd apps/frontend-manage` and `npm install -E react-colorful`, or run `npm i -E -w frontend-manage react-colorful` (or something like this, not sure, but -w specified which workspace to install the package in).